lib/features/mailbox_dashboard/presentation/widgets/app_dashboard/app_grid_dashboard_icon.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:core/presentation/resources/image_paths.dart';
+import 'package:core/presentation/views/button/tmail_button_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_portal/flutter_portal.dart';
+import 'package:get/get.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/app_grid_dashboard_controller.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/widgets/app_dashboard/app_grid_dashboard_overlay.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+class AppGridDashboardIcon extends StatelessWidget {
```

Comment:
Should this widget extend `GetWidget<AppGridDashboardController>` instead?

Reply:
AppGridDashboardController is not a GetLifeCycleBase